sagemaker_neo_compilation_jobs/gluoncv_ssd_mobilenet/gluoncv_ssd_mobilenet_neo.ipynb

```diff
@@ -4,18 +4,18 @@
    "cell_type": "markdown",
```

Comment:
<p>Change</p><p>We then need an S3 bucket that would used for training, and storing the tranied model artifacts. </p><p>To</p><p>We then need an S3 bucket that would be used for training, and storing the tranied model artifacts. </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/1625/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
done